docs/patrol/new-native-setup.mdx

```diff
@@ -0,0 +1,310 @@
+---
+title: Patrol - setup
+---
+
+# Native automation setup
+
+Using Patrol's native automation feature requires more setup work than using
+custom finders. Unfortunately it's necessary, because you have to integrate your
+app with Android and/or iOS native testing tools.
+
+We believe that the powerful features you'll get will make up for it!
+
+<Warning>
+        In this tutorial, we are using example app, which has package name
+        `com.example.myapp` on Android, bundle id `com.example.MyApp` on iOS
+        and `My App` name on both Android and iOS. Replace any occurences
+        of those names with proper values.
+</Warning>
+
+### Add dependency on patrol
+
+If you haven't already, add a dependency on the `patrol` package in the
+`dev_dependencies` section of **pubspec.yaml**.
+
+```
+flutter pub add patrol --dev
+```
+
+### Configure Patrol in pubspec.yaml
+
+Create `patrol` section in your **pubspec.yaml**:
+
+```yaml title="pubspec.yaml"
+dependencies:
+  # ...
+
+dev_dependencies:
+  # ...
+
+patrol:
+  app_name: My App
+  android:
+    package_name: com.example.myapp
+  ios:
+    bundle_id: com.example.MyApp
+```
+
+### Install patrol_cli
+
+Patrol CLI (command-line interface) is a small program that enables running
+Patrol UI tests. It handles the complexity of native iOS and Android testing
+tools itself, so you can focus on writing your tests instead of learning yet
+another platform-specific tool.
+
+1. Install `patrol_cli` executable:
+
+   ```
+   dart pub global activate patrol_cli
+   ```
+
+2. Verify that installation was successful and your enviroment is set up properly:
+
+   ```
+   patrol doctor
+   ```
+
+### Integrate with native side
+
+The 3 first steps were common across platforms. The rest is platform-specific.
+
+Psst... Android is a bit easier to set up, so we recommend starting with it!
+
+<Tabs 
+  defaultValue="android" 
+  values={[
+    { label: "Android", value: "android" },
+    { label: "iOS", value: "ios" },
+  ]}>
+  <TabItem value="android">
+
+      Go to **android/app/src/androidTest/java/com/example/myapp/** in your project
+      directory. If there are no such folders, create them.
+
+      Create a file named `MainActivityTest.java` and copy there the code below.
+
+      ```java title="MainActivityTest.java"
+      package com.example.myapp; // replace "com.example.myapp" with your app's package
```

Comment:
Maybe it's good to mention that user needs to replace it in the warning tile and with additional info where he can find app's package name

Reply:
This info can be easily found by googling, I think it shouldn't be there